cmd/flagutils/utils.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package flagutils
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/cli-utils/pkg/inventory"
+)
+
+func ConvertInventoryPolicy(policy string) (inventory.InventoryPolicy, error) {
```

Comment:
Should there be a simple test for this new utility function?

Reply:
Done